frontend/pages/g/_groupSlug/r/create/plaintext.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <v-form ref="domUrlForm" @submit.prevent="createRecipe">
+      <div>
+        <v-card-title class="headline">{{ $t('recipe.create-recipe-from-text') }}</v-card-title>
+        <v-card-text>
+          <p>{{ $t('recipe.create-recipe-from-text-description') }}</p>
+          <v-container class="pa-0">
+            <v-row>
+              <v-col cols="12">
+                <v-textarea
+                  v-model="recipeText"
+                  :label="$t('recipe.enter-recipe-text')"
+                  :placeholder="$t('recipe.recipe-text-placeholder')"
+                  outlined
+                  auto-grow
+                  rows="6"
+                  :disabled="loading"
+                ></v-textarea>
+              </v-col>
+            </v-row>
+          </v-container>
+        </v-card-text>
+        <v-card-actions v-if="recipeText">
+          <div>
+            <p style="width: 250px">
+              <BaseButton rounded block type="submit" :loading="loading" />
+            </p>
+            <p>
+              <v-checkbox
+                v-model="shouldTranslate"
+                hide-details
+                :label="$t('recipe.should-translate-description')"
+                :disabled="loading"
+              />
+            </p>
+            <p v-if="loading" class="mb-0">
+              {{ $t('recipe.please-wait-processing') }}
+            </p>
```

Comment:
You can probably get rid of this since the submit button already indicates that it's loading